packages/reporters/cli/src/index.js

```diff
@@ -3,6 +3,7 @@
 // $FlowFixMe This is fine.
 const isTTY = process.stdout.isTTY;
 
-export default (isTTY
+export default require('./SimpleCLIReporter')
```

Comment:
For some reason the CLIReporter isn't working anymore in this branch.

Probably has to do with my added transformEvent